<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>S2 Dictionary</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QLineEdit" name="opStatus">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1" rowspan="2">
     <widget class="QTextEdit" name="txtMeaning">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="btnSearch">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLineEdit" name="inpWord">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>Enter a word to search</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QListView" name="listWords"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
